src/mds/objects.py

```diff
@@ -226,6 +226,161 @@ async def create_object_for_id(
     return JSONResponse(response, HTTP_201_CREATED)
 
 
+@mod.get("/objects")
+async def get_objects(
+    request: Request,
+    data: bool = Query(
+        True,
+        description="Switch to returning a list of GUIDs (false), "
```

Comment:
```suggestion
        description="Switch to return a list of GUIDs (false), "
```
At least, I think that's right? 😅 🔤